script/after_success

```diff
@@ -4,7 +4,7 @@ set -e
 branch=$TRAVIS_PULL_REQUEST_BRANCH
 event=$TRAVIS_EVENT_TYPE
 
-# only publish canary releases on PRs against dev
+# only publish canary releases on non release or master branches
```

Comment:
Wouldn't we want this to only publish canary releases _on_ PRs against a release branch?